test/testvm.py

```diff
@@ -943,9 +943,7 @@ def _cleanup(self, quick=False):
     def kill(self):
         # stop system immediately, with potential data loss
         # to shutdown gracefully, use shutdown()
-        if hasattr(self, '_domain') and self._domain:
-            if self._domain.isActive():
-                self._domain.destroy()
+        # we just delete the variables and let libvirt clean up for us
```

Comment:
Won't this further exacerbate the issues with the IPA VM racing with its own shutdown?